src/liquid/components/ld-notification/ld-notification.css

```diff
@@ -1,4 +1,19 @@
 :host {
+  /* layout */
+  --ld-notification-min-height: var(--ld-sp-40);
+  --ld-notification-border-radius: var(--ld-br-m);
+  --ld-notification-inset-y: var(--ld-sp-32);
+
+  /* colors */
+  --ld-notification-col: var(--ld-col-wht);
+  --ld-notification-warn-col: var(--ld-col-neutral);
+  --ld-notification-alert-col: var(--ld-col-wht);
```

Comment:
```suggestion
  --ld-notification-text-col: var(--ld-col-wht);
  --ld-notification-warn-text-col: var(--ld-col-neutral);
  --ld-notification-alert-text-col: var(--ld-col-wht);
```

Reply:
I'll rather keep it as it is. In css it's either background-color or color. There is no text-color. So I'll stick to this.